javascript/javascript3/week3/shopping/shopping.js

```diff
@@ -0,0 +1,280 @@
+// @ts-nocheck
+const mainUL = document.querySelector("#container");
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  async convertToCurrency(currency) {
+    let isCurrencyFound = false;
+    let price;
+    await fetch("https://api.exchangeratesapi.io/latest?base=DKK")
```

Comment:
don't mix await and then, it's confusing. Best practice is to use async / await, and avoid .then(). 
Rewrite this code in the async / await way and you'll see how much easier it is to read.